bin.src/generateCoefficients.py

```diff
@@ -34,11 +35,11 @@
 
     # Parse orbit file input values.
     if args.orbitFile is None:
-        print "Must specify orbit file to use."
+        print("Must specify orbit file to use.")
         exit()
 
     if not os.path.isfile(args.orbitFile):
-        print "Could not find orbit file %s" %(args.orbitFile)
+        print("Could not find orbit file %s" % (args.orbitFile))
     if args.orbitFile.lower().endswith('s3m'):
         skiprows = 1
```

Comment:
I'll have to have a look at what pandas does (because behind the scenes, it's pandas reading the data). Usually pandas assumes the first line that it's supposed to read is the line with the header information. In fact, many of the s3m and des files have different formats and different header formats. 
Often straight up (or very old) s3m files start with a single line starting with "#"
Newer s3m files start with two lines, starting with !!
And some s3m files start with a single line starting with !! ObjId (rather than !!ObjID) ... which again, breaks pandas.

We may just need to do by-hand sanitizing of the input files, at some point, because we also do need to read the header line ... sometimes the files are q/e/i ... and sometimes they're a/e/i.